src/test/java/com/firebolt/jdbc/resultset/FireboltResultSetTest.java

```diff
@@ -519,18 +518,26 @@ void shouldReturnBytes() throws SQLException, IOException {
 		inputStream = getInputStreamWithCommonResponseExample();
 		resultSet = createResultSet(inputStream);
 		resultSet.next();
-		byte[] expected = "Taylor\\'s Prime Steak House".getBytes();
-		assertArrayEquals(expected, resultSet.getBytes(3));
-		assertArrayEquals(expected, resultSet.getBytes("name"));
-		assertArrayEquals(expected, resultSet.getBinaryStream(3).readAllBytes());
-		assertArrayEquals(expected, resultSet.getBinaryStream("name").readAllBytes());
-		assertArrayEquals(expected, resultSet.getObject(3, byte[].class));
-		assertArrayEquals(expected, resultSet.getObject("name", byte[].class));
-		assertArrayEquals(expected, resultSet.getBlob(3).getBinaryStream().readAllBytes());
-		assertArrayEquals(expected, resultSet.getBlob("name").getBinaryStream().readAllBytes());
+		String expectedString = "Taylor\'s Prime Steak House";
+		assertEquals(expectedString, resultSet.getString(3));
+		assertEquals(expectedString, resultSet.getString("name"));
+		assertEquals(expectedString, readAll(resultSet.getCharacterStream(3)));
+		assertEquals(expectedString, readAll(resultSet.getCharacterStream("name")));
+		assertEquals(expectedString, resultSet.getObject(3, String.class));
+		assertEquals(expectedString, resultSet.getObject("name", String.class));
 		resultSet.next();
-		assertNull(resultSet.getBytes(3));
-		assertNull(resultSet.getBlob(3));
+		assertNull(resultSet.getString(3));
+		assertNull(resultSet.getString(3));
+	}
```

Comment:
This test is `shouldReturnBytes` but we're not testing bytes anymore. If this is already tested elsewhere we can remove it?